ui/dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:stable
+
+RUN rm /etc/nginx/conf.d/default.conf
+
+COPY nginx.conf /etc/nginx/conf.d
+
+COPY dist /usr/share/nginx/html
+
+EXPOSE 80
```

Comment:
Because we will be using 8080 or the PORT environment variable (see https://github.com/reearth/reearth-flow/pull/243#discussion_r1635528967 for details), we can remove this!

Actually, EXPOSE does nothing 😎

> The EXPOSE instruction doesn't actually publish the port. It functions as a type of documentation 

[EXPOSE, Dockerfile](https://docs.docker.com/reference/dockerfile/)

Reply:
Done.
Ref: https://github.com/reearth/reearth-flow/pull/243/commits/959a3b4e9cb4803be4b7d3c3decff5e1410db2ed